tests/telemetry/events/event_utils.py

```diff
@@ -45,3 +48,29 @@ def restart_monit(duthost):
 def check_monit_running(duthost):
     monit_services_status = duthost.get_monit_services_status()
     return monit_services_status
+
+
+def create_ip_file(duthost, data_dir, json_file, start_idx, end_idx):
+    ip_file = os.path.join(data_dir, json_file)
+    with open(ip_file, "w") as f:
+        for i in range(start_idx, end_idx + 1):
+            json_string = f'{{"test-event-source:test": {{"test_key": "test_val_{i}"}}}}'
+            f.write(json_string + '\n')
+    dest = "~/" + json_file
+    duthost.copy(src=ip_file, dest=dest)
+
+
+def event_publish_tool(duthost, json_file):
+    ret = duthost.shell("python ~/events_publish_tool.py -f ~/{}".format(json_file))
```

Comment:
>events_publish_tool [](http://example.com/codeflow?start=34&length=19)

Where does this py file coming from?

Reply:
Part of eventd in image: https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-eventd/tools/events_publish_tool.py

Reply:
I see the py file copied from eventd container to the host. Is it possible keep it in original eventd container, and just `docker exec` ?

Reply:
Yes, no longer copy py file from eventd to host, but instead utilize docker exec